### PR TITLE
feat: auto deploy new container builds

### PR DESCRIPTION
**Pull request by WillOnGit**
Marking as draft until related auto deploy workflow is complete in `infrastructure`

### CODE REVIEW

**Comment by WillOnGit** [comment]
The `infrastructure` workflow is ready, so this can be merged after review now.

**Review by Mearman — APPROVED** [review]

File: .github/workflows/infrastructure-pr.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+jobs:
+  main:
+    name: Trigger workflow_dispatch event in infrastructure repo
+    runs-on: ubuntu-latest
+
+    steps:
+      # TODO: review Github CLI
+      - name: call workflow_dispatch API
+        run: |
+          curl -L \
```
Comment (cdata) [inline .github/workflows/infrastructure-pr.yml:27]: Thoughts on using `gh` to do this? Is it possible?

Comment (Mearman) [reply .github/workflows/infrastructure-pr.yml:27]: That's what the TODO is for. Just this version for know as we know it works.

**Review by cdata — APPROVED** [review]
Woot, LGTM!

**Pull request merged**